### ISSUE TITLE
no module named startcast

### ISSUE THREAD

**Issue by brianwhigham** [diequeiroz/start-chromecast-CLI#1]
First of all, I'm a sysadmin, not a python programmer.  So, I'll start with this braindead plea for help:

I run **startcast myFriendlyName 5F333556** and get this output:

```
Traceback (most recent call last):
  File "/usr/local/bin/startcast", line 9, in <module>
    load_entry_point('startcast==0.1.0', 'console_scripts', 'startcast')()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 521, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2632, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2312, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2318, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named startcast
```

Having simply installed with "sudo pip install startcast", pip puts things in /usr/local/lib/python2.7/dist-packages/startcast-0.1.0.egg-info on my Ubuntu 15.04 box.  On my mac, it goes in /Library/Python/2.7/site-packages/startcast-0.1.0.egg-info.  I do have the /usr/local/bin/startcast script on both machines.

Maybe I'm missing something; but, shouldn't I find startcast.py someplace?  It doesn't seem to be included as part of the pip install.

FWIW, here are my sys.path outputs:

Ubuntu:

```
['', '/usr/local/lib/python2.7/dist-packages/esptool-0.1.0-py2.7.egg', '/usr/lib/python2.7/dist-packages', '/usr/lib/python2.7', '/usr/lib/python2.7/plat-x86_64-linux-gnu', '/usr/lib/python2.7/lib-tk', '/usr/lib/python2.7/lib-old', '/usr/lib/python2.7/lib-dynload', '/home/oobx/.local/lib/python2.7/site-packages', '/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages/PILcompat', '/usr/lib/python2.7/dist-packages/gtk-2.0', '/usr/lib/pymodules/python2.7', '/usr/lib/python2.7/dist-packages/ubuntu-sso-client', '/usr/lib/python2.7/dist-packages/wx-3.0-gtk2']
```

mac:

```
['', '/Library/Python/2.7/site-packages/pip-7.1.2-py2.7.egg', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python27.zip', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-darwin', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/plat-mac/lib-scriptpackages', '/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-tk', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-old', '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/lib-dynload', '/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC', '/Library/Python/2.7/site-packages']
```

**Comment by diequeiroz** [comment]
@brianwhigham is this issue resolved on #2?

**Comment by brianwhigham** [comment]
Now my pip is hosed on Linux, complaining about "cannot import name
IncompleteRead".  So, I'll focus on the Mac, which still has the problem.

```
Traceback (most recent call last):
  File "/usr/local/bin/startcast", line 9, in <module>
    load_entry_point('startcast==0.1.0', 'console_scripts',
```

'startcast')()
      File
"/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py",
line 357, in load_entry_point
        return get_distribution(dist).load_entry_point(group, name)
      File
"/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py",
line 2394, in load_entry_point
        return ep.load()
      File
"/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py",
line 2108, in load
        entry = **import**(self.module_name, globals(),globals(),
['**name**'])
    ImportError: No module named startcast

**Comment by brianwhigham** [comment]
I don't know how pip broke at such a terrible time.  But I fixed it using
this article
http://stackoverflow.com/questions/27341064/how-do-i-fix-importerror-cannot-import-name-incompleteread.
Basically, uninstalled Ubuntu pip and installed pip with easy_install -U
pip.

Now sudo pip install startcast on my Ubuntu gives the same error as the Mac.

Shouldn't the timestamp on https://pypi.python.org/pypi/startcast be
updated?  Is it not best to bump the version?

Thanks

**Comment by diequeiroz** [comment]
@brianwhigham the new version is not published yet.

**Comment by brianwhigham** [comment]
Well, it seems to work with "python setup from my repo clone".  I don't
know how to test that it will continue to work once it's published.  I'm
not clear on the publishing process and what gets pushed up to pypi.

**Comment by diequeiroz** [comment]
Version 0.1.1 is published now!

**Comment by brianwhigham** [comment]
now works on Ubuntu and Mac.

Thanks so much

**Comment by diequeiroz** [comment]
fixed on #2
